ts/components/RemindEmailValidationOverlay.tsx

```diff
@@ -0,0 +1,160 @@
+  }
+
+  public componentDidUpdate(prevprops: Props, prevstate: State) {
+    const { isValidEmail } = this.props;
+    const { dispatched } = this.state;
+    if (
+      !prevprops.isValidEmail &&
+      !isValidEmail &&
+      prevstate.dispatched &&
+      dispatched
+    ) {
+      this.setState(
+        {
+          dispatched: false
+        },
+        this.props.navigateBack
+      );
+    }
+  }
+
+  public render() {
+    const { optionProfile } = this.props;
+    const profileEmail = optionProfile
+      .map(_ => untag(_.spid_email))
+      .getOrElse("");
+    return (
+      <TopScreenComponent
+        customRightIcon={{
+          iconName: "io-close",
+          onPress: this.props.navigateBack
+        }}
+      >
+        <Content>
+          <Image
+            style={{ alignSelf: "center" }}
+            source={require("../../img/email-checked-icon.png")}
+          />
+          <View spacer={true} extralarge={true} />
+          <H2 style={{ textAlign: "center" }}>
+            {I18n.t("reminders.email.title")}
+          </H2>
+          <View spacer={true} />
+          <Text>
+            {I18n.t("reminders.email.modal1")}
+            <Text bold={true}>{` ${profileEmail}: `}</Text>
+            {I18n.t("reminders.email.modal2")}
+          </Text>
+          <View spacer={true} />
+          <Button
+            block={true}
+            light={true}
+            bordered={true}
+            onPress={() => {
+              // https://www.pivotaltracker.com/story/show/168247365
+              Alert.alert(I18n.t("global.notImplemented"));
+            }}
+          >
+            <Text>{I18n.t("reminders.email.button1")}</Text>
+          </Button>
+        </Content>
+
+        <FooterWithButtons
+          type={"TwoButtonsInlineThirdInverted"}
+          leftButton={{
+            block: true,
+            bordered: true,
+            onPress: () => {
+              this.setState(
+                {
+                  dispatched: false
+                },
+                this.props.navigateToEmailInsertScreen
+              );
+            },
+            title: I18n.t("reminders.email.button2")
+          }}
+          rightButton={{
+            block: true,
+            primary: true,
+            onPress: () => {
+              this.setState(
+                {
+                  dispatched: true
+                },
+                this.props.validateEmail
+              );
+            },
+            title: I18n.t("global.buttons.ok")
+          }}
+        />
+      </TopScreenComponent>
+    );
+  }
+}
+
+// tslint:disable-next-line: variable-name
+const mapStateToProps = (state: GlobalState) => ({
+  optionProfile: pot.toOption(state.profile),
+  isValidEmail: !isEmailEditingAndValidationEnabled && !!state // TODO: get the proper isValidEmail from store
+});
+
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  navigateBack: () => dispatch(navigateBack()),
```

Comment:
Since this will be shown as modal, navigation back is not valid.
To hide this modal, I guess you have to add a property (e.g **_onClose_**) which you fire when the x button is pressed.
Then the parent **_withValidatedEmail.tsx_** will provide to dispose the modal.